### DISCUSSION TITLE
[09/28/23] Mojo Update (0.3.1) targeting installation issues

### DISCUSSION THREAD

**Discussion by ematejska** [modular/modular#944] [Announcements]
Hey everyone 👋 ,

We just released a patch update of Mojo 🔥(v0.3.1) targeting installation issues.  For those of you experiencing installation issues on low core machines or failures to install dependencies, give it a try.

To see what's changed, [check out the changelog](https://docs.modular.com/mojo/changelog.html).

Ensure you have the latest release of the modular CLI tool and then update your version of the Mojo SDK:

```bash
$ sudo apt-get update && sudo apt-get install modular
$ modular clean
$ modular install mojo
```

Please note:
* Ensure to run the clean steps above.
* Get started with Mojo at [https://www.modular.com/mojo](http://www.modular.com/mojo), download our [VS code extension](https://marketplace.visualstudio.com/items?itemName=modular-mojotools.vscode-mojo), and see our [Mojo examples in Github](https://github.com/modularml/mojo/tree/main/examples)!

Cheers! 🍻

**Comment by xiangnanscu** [comment]
ubuntu 20.04, after ` sudo apt-get update `:
```
root@win10:~/jaqn#  apt-get install modular
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package modular
```

**Reply by Sharktheone** [reply]
You need to run the installation script, you can find it on the Modular website. Modular isn't available in the standard Ubuntu / Debian repos, the installation script adds this repository.

**Reply by ematejska** [reply]
Were you able to resolve this issue with the instructions above?  Make sure to have the CLI tool installed at least once per the instructions at: https://developer.modular.com/download and then you can use the commands above.
